doc/content/guides/connecting-gateways/thethingsgateway/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "The Things Gateway"
+description: ""
+weight: 2
+menu:
+  main:
+    weight: 2
+---
+
+The Things Gateway is an MQTT based single channel LoRaWAN gateway. The technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/gateway/). This page guides you to connect it to a The Things Stack instance.
```

Comment:
- The Things Gateway is not single-channel
- connect it _to The Things Stack_

Reply:
- No need to mention it's MQTT here
- More clear name pending, see out-of-band comms